src/api-guides/travel/hotelservice2/hotel-shop-book-cancel.markdown

```diff
@@ -0,0 +1,193 @@
+
+6. Upon selecting View Rooms, the UI displays all available rates for the chosen hotel. 
+
+    <a href='./images/general-walkthrough/hotel-rate-listing.png'><img style="max-width:300px" src="./images/general-walkthrough/hotel-rate-listing.png"/></a>
+
+7. Note each rate displays a Rules and cancellation policy link. When the business traveler clicks the link, if the cancellaton details were defered as indicated by [TPA_Extensions/RateDetailsInd](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Availability.html#res-schema), a [RateDetails](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Rate-details.html) request will be invoked. The response content will be displayed in a pop-up window.
+
+    <a href='./images/general-walkthrough/rules-and-cancellation-policy.png'><img style="max-width:300px" src="./images/general-walkthrough/rules-and-cancellation-policy.png"/></a>
+
+    [Sample rate details request](./sample-requests/general-walkthrough/ratedetails-rq.xml)
+
+    [Sample rate details response](./sample-requests/general-walkthrough/ratedetails-rs.xml)
+
+    More details about the `RateDetails` requests [here](./faq#availability)
+
+8. The UI displays all available rates for the chosen hotel. The Business traveler selects the top most rate. The Trip Summary page is displayed where the Business traveler can set the Hotel Preferences, Enter Guest information (from their profile), select the method of payment and view the total estimated price. The Rate rules and cancellation policy will be displayed as well. Like the previous step, if the cancellation details were deferred, a [RateDetails](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Rate-details.html) request will be invoked.
+
+9. The business traveler agrees to the rate rules, restrictions and cancellation policy and clicks Reserve Hotel and Continue.
+
+    <a href='./images/general-walkthrough/review-and-reserve-hotel.png'><img style="max-width:300px" src="./images/general-walkthrough/review-and-reserve-hotel.png"/></a>
+
+
+10. When the reservation request succeeds, the details of the reservation will be displayed to the business traveler
+
+    <a href='./images/general-walkthrough/trip-confirmation.png'><img style="max-width:300px" src="./images/general-walkthrough/trip-confirmation.png"/></a>
+
+    [Sample book request](./sample-requests/general-walkthrough/book-rq.xml)
+
+    [Sample book response](./sample-requests/general-walkthrough/book-rs.xml)
+
+    More details about the `Reservation` requests [here](./faq#reservation)
+
+11. When the reservation details are displayed SAP Concur will invoke a [ReadItinerary](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Read-Itinerary.html) request. If a Read request does not arrive within 5 minutes for a given Reservation, then the supplier should treat that Reservation as an orphan and should thus seek to cancel it.
```

Comment:
```suggestion
### Hotel Cancellation
11. When the reservation details are displayed SAP Concur will invoke a [ReadItinerary](https://developer.concur.com/api-reference/direct-connects/hotel-service-2/Read-Itinerary.html) request. If a Read request does not arrive within 5 minutes for a given Reservation, then the supplier should treat that Reservation as an orphan and should thus seek to cancel it.
```

Reply:
The Hotel Cancellation header would fit better at step 12. Step 11 is an action that is performed without user interaction, and is intended to be an acknowledgement we have received the booking response.